annotation-processor/src/main/java/org/hibernate/validator/ap/AbstractElementVisitor.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.ap;
+
+import java.util.Collection;
+import java.util.Set;
+import javax.lang.model.util.ElementKindVisitor6;
+
+import org.hibernate.validator.ap.checks.ConstraintCheckIssue;
+import org.hibernate.validator.ap.util.CollectionHelper;
+import org.hibernate.validator.ap.util.Configuration;
+import org.hibernate.validator.ap.util.MessagerAdapter;
+
+/**
+ * An abstract {@link javax.lang.model.element.ElementVisitor} that should be used for implementation
```

Comment:
No biggie, but we can use simple names in JavaDocs (and import them), too. Reads a tad nicer.